### PR TITLE
[FLINK-3903][docs] adding alternative installation for OSX users

### PR DESCRIPTION
**Pull request by heytitle**
# Changes

## Screenshot of setup guideline for stable version
screenshot 2017-02-24 17 42 34
screenshot 2017-02-24 17 42 40

## Screenshot of setup guideline  for snapshot version
screenshot 2017-02-24 17 41 08

### CODE REVIEW

File: docs/quickstart/setup_quickstart.md [hunk 1]
```diff
@@ -72,6 +72,15 @@ $ cd build-target               # this is where Flink is installed to
 ~~~
 {% endif %}
 
+### Alternatively
```
Comment (greghogan) [inline docs/quickstart/setup_quickstart.md]: What if we renamed "Alternatively" to "Install on MacOS".

And perhaps pull the prior "Download" into the if/else and rename to "Download and Unpack" and "Download and Compile"?

Comment (heytitle) [inline docs/quickstart/setup_quickstart.md]: Should we also put to OSX installation into the if/else?

Comment (greghogan) [inline docs/quickstart/setup_quickstart.md]: Yes. And perhaps use the `<div class="codetabs" markdown="1">` formatting as with `dataset_transformation.md` and elsewhere. With tabs it is easy to see the different choices.

**Comment by uce** [comment]
Looks good, merging.

**Pull request closed without merging**